### PR TITLE
Repeater validation fixes

### PR DESCRIPTION
**Pull request by shuwen**
makes custom validation always win, adds small message style for elements below a width threshold, and adds undocumented flag to suppress messages altogether

@anthonykoerber @dlasky @jcmoore

### CODE REVIEW

File: test/strand-repeater-row.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+					name: "This is an error",
+					favColor: "This is an error"
+				});
+
+				flush(function() {
+					var messages = Polymer.dom(row.root).querySelectorAll('strand-form-message'),
+						areMsgsSmall = Array.prototype.slice.call(messages)
+							.map(function(node) { return node.layout === 'small' });
+
+					expect(areMsgsSmall).to.contain(true);
```
Comment (dlasky) [inline test/strand-repeater-row.html]: shouldn't we know specifically which is which here?

**Comment by dlasky** [comment]
👍

File: src/shared/behaviors/validatable.html [hunk 2]
```diff
@@ -63,7 +63,7 @@
 				});
 				return result.length === this.testSet.length;
 			} else {
-				return false;
+				return true;
```
Comment (jcmoore) [inline src/shared/behaviors/validatable.html]: how exposed is this API?  maybe consider changing the method name to `isValid` or `isValueValid` (somewhat less ambiguous about what the result/return-value means) if it won't require a full point-release due to breaking interface.

**Comment by anthonykoerber** [comment]
IMHO - all `strand-inline-message`s should be set to `layout=small`, just to have some visual consistency... also  `layout=small` should probably be `size=small` but this is debatable (if this is an internal-only feature).

otherwise LGTM 👍

File: src/strand-grid/index.html [hunk 3]
```diff
@@ -3,12 +3,6 @@
 <head>
```
Comment (dlasky) [inline src/strand-grid/index.html]: why is this comitted?

File: src/strand-repeater-row/strand-repeater-row.scss [hunk 4]
```diff
@@ -52,19 +52,18 @@
...
+}
+
+@keyframes fadeIn {
+	from { opacity: 0; }
+	to { opacity: 1; }
 }
 
 .error-message[visible] {
-	animation: 300ms rollIn ease-out;
+	animation: 5s rollDown ease-out, 300ms fadeIn ease-out;
```
Comment (dlasky) [inline src/strand-repeater-row/strand-repeater-row.scss]: are you sure this works in 52?

**Comment by dlasky** [comment]
removing that index change, otherwise new patchset looks good to me.

**Pull request merged**